buildprebid.sh

```diff
@@ -81,19 +85,18 @@ mkdir $TEMPDIR
 ###########################
 echoX "Run unit tests"
 cd $LIBDIR
-(./gradlew -i clean test > $LOGPATH/testResults.log 2>&1) || (echoX "Unit tests failed, check log in $LOGPATH/testResults.log" && echoX "End Script" && exit 1 ) &
+(./gradlew -i --no-daemon clean test > $LOGPATH/testResults.log 2>&1) || (die "Unit tests failed, check log in $LOGPATH/testResults.log") &
 PID=$!
 spinner $PID &
 wait $PID
 
 echoX "Assemble builds"
 cd $LIBDIR
 # clean existing build results, exclude test task, and assemble new release build
-(./gradlew -i -x test build > $LOGPATH/build.log 2>&1 || { echoX "Build failed, check log in $LOGPATH/build.log"; echoX "End Script"; exit 1; } ) &
+(./gradlew -i --no-daemon -x test build > $LOGPATH/build.log 2>&1 || die "Build failed, check log in $LOGPATH/build.log" ) &
 PID=$!
 spinner $PID &
 wait $PID
-echoX "Build succeeded"
```

Comment:
This line is a bit redundant here. If it fails, then this line will never be run, if the script succeeds and continues there're following lines indicate that.